Sample.java

```diff
@@ -1,7 +1,124 @@
 
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Three line explanation of solution in plain english
+class Sample {
 
-// Your code here along with comments explaining your approach
+    // Time Complexity : O(N)
+    // Space Complexity : o(1)
+    // Did this code successfully run on Leetcode : Yes
+
+    // Problem 1
+    /**
+     * 1. Take 2 pointers. i and j.
+     * 2. j keeps track of all the elements and i is for keeping track of at max 2 duplicates in an array.
+     * 3. value of count is checked if it is not more than 2.
+     */
+
+    public int removeDuplicates(int[] nums) {
+        
+        if(nums == null || nums.length == 0)
+            return 0;
+        
+        int i = 1, j = 1;
+        int count = 1;
+        
+        while(j < nums.length) {
+            if(nums[j] != nums[j-1]) {
+                nums[i] = nums[j];
+                i++;
+                count = 1;
+            }
+            else {
+                if(count < 2) {
+                    nums[i] = nums[j];
+                    i++;
+                    count++;
+                }
+            }
+            j++;
+        }
+        
+        return i;
+        
+    }
+    
+    // Time Complexity : O(N)

```

Comment:
Two array sizes are M and N, we are traversing through both. So, complexity has to be O(M+N) here

Reply:
We are traversing both arrays simultaneously, so it should be O(N) where N is length of first array.

Reply:
Then, that is right. I thought N as actual size of first array without considering space for second array's elements.